index.js

```diff
@@ -5,19 +5,29 @@ import element from 'magic-virtual-element';
 let buttonIndex = 0;
 
 export function render ({ props }) {
-  const fn = 'youtubeSubscribeButtonEvent' + buttonIndex;
-  global[fn] = function (payload) {
-    if (payload.eventType === 'subscribe' && props.onSubscribe) {
-      props.onSubscribe();
-    }
-
-    if (payload.eventType === 'unsubscribe' && props.onUnsubscribe) {
-      props.onUnsubscribe();
-    }
-  };
-
-  const id = '__deku-yt-subscribe-wrapper-' + buttonIndex;
-  buttonIndex++;
+  let fn = '';
```

Comment:
i know this is kind of separate from fixing the problem, but it seems like this variable should be called `fnName` since it's a string and not a function